static/index.js

```diff
@@ -0,0 +1,136 @@
+let reviewsList = [
+  {
+    review: "Cool, cool, cool, cool. No doubt, no doubt, no doubt.",
+    rating: 5,
+  },
+];
+
+function saveToLocalStorage(reviews) {
+  const reviewsString = JSON.stringify(reviews);
+  localStorage.setItem("reviews-list", reviewsString);
+}
+
+function loadFromLocalStorage() {
+  const reviewsString = localStorage.getItem("reviews-list");
+
+  return JSON.parse(reviewsString);
```

Comment:
Super što koristiš f-je za semantički različite operacije 👍 
Ovdje će nastati bug (pogledaj komentar pri dnu) što se dogodi.

Pripazi da return f-je bude "dobra vrijednost". Ovdje si mogao napraviti

Ovdje je problem što direktno vraćaš null ili array reviewova (ovisno o stanju u local storage-u).
Ne koristimo typescript pa nemaš typingse i kod se ne žali i ovakve stvari se dogode. Vidjet češ kad krenemo raditi s typescriptom da je malo lakše uhvatiti ove stvari.

Ovdje si mogao napraviti sljedeće:
Ili `return JSON.parse(reviewsString) || []`; -> osigurati se da f-ja uvijek vrati array (bilo prazan ili s nekim review itemima).
Ili na inicijalni poziv (linija 135):
`reviewsList = loadFromLocalStorage() || [];`